e2e/tests/bulkScanUserUploadsScannedDocuments_test.js

```diff
@@ -6,11 +6,7 @@ let caseId;
 Feature('Uploading bulk scan document');
 
 BeforeSuite(async (I, caseViewPage, submitApplicationEventPage, handleSupplementaryEvidenceEventPage, attachScannedDocsEventPage) => {
```

Comment:
i guess these are not needed now

Reply:
I'd leave it because uploading documents for bulkScan is not covered anywhere else